api/helper/builder/ocm_identity.go

```diff
@@ -10,6 +14,18 @@ func (b *Builder) ExtraIdentity(name string, value string) {
 	b.ocm_meta.ExtraIdentity.Set(name, value)
 }
 
+func (b *Builder) ExtraIdentities(extras ...string) {
+	b.expect(b.ocm_meta, T_OCMMETA)
+
+	id := metav1.NewExtraIdentity(extras...)
+	if b.ocm_meta.ExtraIdentity == nil {
```

Comment:
I just noticed this, but why do all fields not correlate to the idiomatic naming? such as 

```suggestion
	if b.ocmMeta.ExtraIdentity...
```

Reply:
it was intended as replacement for a sub structure like `b.ocm.meta`.